Homework8.medium/class.js

```diff
@@ -0,0 +1,63 @@
+class shopingCart{
+    constructor() {
+        this.items = [];
+        this.totalPrice = 0
+        
+    }
+
+
+    additem(product, price) {
+        this.items.push(product)
+        this.totalPrice = this.totalPrice + price;
+         
+    }
+    removeItem(product, price) {
+       this.items.shift(product)
```

Comment:
with shift we remove the first item from the list. Not the product you give as a parameter. Please use the following algorithm to remove an item:
```javascript
const indexOfItemToBeRemoved = this.items.findIndex(item => item === product); // to get the index of the item that needs to be removed
this.items.splice(indexOfItemToBeRemoved, 1); // to remove the item
```
the reduction of the price is correct 😄 
Please read more on 
* https://www.w3schools.com/jsref/jsref_splice.asp
* https://www.w3schools.com/jsref/jsref_findindex.asp